features/dd-sdk-android-session-replay-compose/consumer-rules.pro

```diff
@@ -0,0 +1,2 @@
+# Keep the Compose internals class name. We need this in the SR recorder.
+-keepnames class androidx.appcompat.widget.DropDownListView
```

Comment:
it seems we should have more classes here by looking on the content of `ComposeReflection`